runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -11818,9 +11817,17 @@ TR::Register *J9::Z::TreeEvaluator::inlineCheckAssignableFromEvaluator(TR::Node
    generateS390BranchInstruction(cg, TR::InstOpCode::BRC, TR::InstOpCode::COND_BRC, node, doneLabel); // exit OOL section
    outlinedSlowPath->swapInstructionListsWithCompilation();
 
+   generateS390LabelInstruction(cg, TR::InstOpCode::label, node, successLabel, deps);
```

Comment:
Register dependency should be attached to the end of internal control flow label. We do not need it here.